test/new_relic/agent/stats_engine_test.rb

```diff
@@ -178,22 +178,54 @@ def test_record_scoped_and_unscoped_metrics_records_multiple_unscoped_metrics
     ])
   end
 
-  def test_record_scoped_and_unscoped_metrics_is_thread_safe
+  def test_record_scoped_and_unscoped_metrics_is_thread_safe_without_traced_threads
     threads = []
     nthreads = 25
     iterations = 100
 
-    nthreads.times do |tid|
-      threads << Thread.new do
-        iterations.times do
-          in_transaction('txn') do
-            @engine.tl_record_scoped_and_unscoped_metrics('m1', ['m3'], 1)
-            @engine.tl_record_scoped_and_unscoped_metrics('m2', ['m4'], 1)
+    with_config(:'instrumentation.thread.tracing' => false) do
+      nthreads.times do |tid|
+        threads << Thread.new do
+          iterations.times do
+            in_transaction('txn') do
```

Comment:
This reminds me, should we write somewhere that it's no longer supported to create transactions inside of threads if `instrumentation.thread.tracing` is true?

Reply:
I haven't seen anything in our documentation that says that we definitely do support it, or any recommendations for doing it. If you have seen anything like that we should definitely update that, but I don't think people are actually doing this very often, it's probably pretty rare (if anyone is doing it), so it might just be confusing for most people

Reply:
No need to take action then! :)